### PR TITLE
kbd: 2.0.3 -> 2.0.4

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change
The unicode_{start,stop} scripts fail due to a hardcoded path.
I patched that and also updated the package.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using (nope, it's a mass rebuild)
- [x] Tested execution of all binary files
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by rnhmjoj** [comment]
bump

File: pkgs/os-specific/linux/kbd/default.nix [hunk 1]
```diff
@@ -1,8 +1,11 @@
-{ stdenv, fetchurl, autoreconfHook, gzip, bzip2, pkgconfig, flex, check, pam }:
+{ stdenv, fetchurl, autoreconfHook,
+  gzip, bzip2, pkgconfig, flex, check,
+  pam, coreutils
+}:
 
 stdenv.mkDerivation rec {
   name = "kbd-${version}";
-  version = "2.0.3";
+  version = "2.0.4";
```
Comment (matthiasbeyer) [inline pkgs/os-specific/linux/kbd/default.nix]: The hash didn't change with the package update? I doubt that! :smile:

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]

File: pkgs/os-specific/linux/kbd/default.nix (hunk 1 above)
Comment (rnhmjoj) [inline pkgs/os-specific/linux/kbd/default.nix]: Oops

**Comment by grahamc** [comment]
Applied in 0577020929

**Comment by grahamc** [comment]
Thank you!

**Comment by rnhmjoj** [comment]
What happened to this change?

**Comment by therealpxc** [comment]
It looks like these changes are not in fact present in master. @matthiasbeyer @grahamc can we reopen this? It includes fixes for a couple scripts which are currently broken (see the referencing pull request above).

**Comment by matthiasbeyer** [comment]
I have no rights to reopen this.

**Comment by rnhmjoj** [comment]
I can make a new one. What is 0577020929 anyway?

**Comment by therealpxc** [comment]
Did you ever make the new PR? I'd like to see your changes merged. I'm tired of everything wanting to rebuild from source! :-)

**Comment by rnhmjoj** [comment]
I was waiting from @grahamc to reply but I guess I'll go on.

**Pull request closed without merging**